docs/remixd.md

```diff
@@ -1,34 +1,35 @@
 Remixd: Access your Local Filesystem 
 =========================================
+
+[![npm version](https://badge.fury.io/js/%40remix-project%2Fremixd.svg)](https://www.npmjs.com/package/@remix-project/remixd)
+
 To give the Remix IDE (the web app) access to a folder on your computer, you need to use **Remixd** - the plugin along with **remixd** - the cli/npm module. 
 
-The **Remixd** plugin can be activated from the plugin manager or in the **File Explorers** - see the image below.  The **connect to local host** - will activate the **Remixd** plugin.
+The **Remixd** plugin can be activated from the plugin manager or in the **File Explorers** - see the image below.  The **connect to localhost** - will activate the **Remixd** plugin.
 
 ![](images/a-remixd-fe.png)
 
-Once you click **connect to local host** or activate Remixd from the **Plugin Manager**, a modal will come up:
+Once you click **connect to localhost** or activate Remixd from the **Plugin Manager**, a modal will come up:
 
 ![](images/a-remixd-modal.png)
 
 The Remixd plugin is a **websocket plugin** and it has no UI other than this modal dialog box - so you won't see a Remixd icon in the icon panel.
 
-Before you hit **Connect**, you need to install the **remixd** NPM module and run the **remixd** command. 
+Before you hit **Connect**, you need to install the [remixd NPM module](https://www.npmjs.com/package/@remix-project/remixd) and run the **remixd** command. 
 
 The code of `remixd` is
 [here](https://github.com/ethereum/remix-project/tree/master/libs/remixd) .
 
 ### remixd Installation
-**remixd** can be globally installed using the following command:
+
+**remixd** is an NPM module and can be globally installed using the following command:
 `npm install -g @remix-project/remixd`
 
 Or just install it in the directory of your choice by removing the -g flag:
 `npm install @remix-project/remixd`
 
-**NOTE:** The npm address as well as the github repo of remixd have changed - in both cases moving under **remix-project**. 
-- **Github** address is:<br>
-**https://github.com/ethereum/remix-project/tree/master/libs/remixd**
-- **NPM** address is: <br>
-**https://www.npmjs.com/package/@remix-project/remixd**
+**NOTE:** When remixd NPM module is installed, it also installs [Slither](https://github.com/crytic/slither), [solc-select](https://github.com/crytic/solc-select#quickstart) and set [solc](https://docs.soliditylang.org/en/latest/installing-solidity.html) to latest version i.e. 0.8.15 currently, provided `Python3.6+ (pip3)` should be already there on the System. (Supported since Remixd `v0.6.3`). In case of any discrepany, Slither can also installed along with other dependencies using command `remixd -i slither`
```

Comment:
This sentence is a bit long and it needs a small grammatical fix.  Try this: 

**NOTE:** When the remixd NPM module is installed, it also installs [Slither](https://github.com/crytic/slither), [solc-select](https://github.com/crytic/solc-select#quickstart) and sets [solc](https://docs.soliditylang.org/en/latest/installing-solidity.html) to latest version i.e. 0.8.15 currently. 

**ALSO NOTE:** `Python3.6+ (pip3)` needs to already be installed on the System. (This packaging of Slither with the remixd module is supported since Remixd `v0.6.3`). In case of any discrepany, Slither can also installed along with other dependencies using command `remixd -i slither`

Reply:
updated